spec/boring_service_spec.rb

```diff
@@ -21,6 +21,18 @@ def first_pre_hook
   end
 end
 
+class MultiTypeService < BoringService
+  parameter :required_param, [String, Integer]
+
+  def call
+    {
+      required_param: @required_param,
```

Comment:
```suggestion
      { required_param: }
```

Any parameter you define will automatically have a attribute reader, so you can call `required_param` (no need for `@`) from within any instance method.

Reply:
to be fair I just wanted to make it look consistent with your other test class a few lines above 🤔 

<img width="502" alt="image" src="https://github.com/user-attachments/assets/2b930e02-221a-447a-a7f0-01bddddca599">